packages/fake-driver/lib/driver.js

```diff
@@ -24,6 +25,27 @@ class FakeDriver extends BaseDriver {
     };
   }
 
+  proxyActive () {
```

Comment:
So this function _doesn't_ actually need a `sessionId`?

Reply:
Nope, one is required in the umbrella driver since it can handle multiple sessions and so disambiguation is required. but an instance of fake driver only ever represents its own session.